RELEASE.md

```diff
@@ -165,34 +165,34 @@ up with a release for `bpfman-operator`.
-      [Generating Release Notes](#generating-release-notes).
-    - Update the
-      [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml)
-      version for the workspace:
-        - `version = "x.y.z"`
-        - `bpfman = { version = "x.y.z", path = "./bpfman" }"`
-        - `bpfman-api = { version = "x.y.z", path = "./bpfman-api" }`
-        - Note: `bpfman-csi` does not need to be updated.
-    - Run `cargo generate-lockfile`
-    - Update the bpfman version in the [bpfman/examples/Makefile][examples-makefile]:
-      - VERSION ?= x.y.z
-    - Add a new `bpfman/examples/config/v0.x.y/` and
-      `bpfman/examples/config/v0.x.y-selinux/` config directory for the release
-      version by copying the latest release directory and running a search for
-      the current release and replace with the new release.
-    - Add new example config directories for any new examples added since the
-      last release.
-    - Update dispatcher tags.
-      - Modify the tag for `XDP_DISPATCHER_IMAGE` and `TC_DISPATCHER_IMAGE` in
-        `bpfman/src/lib.rs` from `latest` to the new release tag.
-      - Manually add the new release tag to the latest version of the following
-        dispatcher images:
-        - https://quay.io/repository/bpfman/xdp-dispatcher
-        - https://quay.io/repository/bpfman/tc-dispatcher
-    - Search the code and docs for the current version number without the "v"
-      (e.g., 0.5.1) and replace it with the new version number where it makes
-      sense.  (Be careful, though, because not all should be replaced.)
+  - Add a new changelog for the release using the process described in
+    [Generating Release Notes](#generating-release-notes).
+  - Update the
+    [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml)
+    version for the workspace:
+      - `version = "x.y.z"`
+      - `bpfman = { version = "x.y.z", path = "./bpfman" }"`
+      - `bpfman-api = { version = "x.y.z", path = "./bpfman-api" }`
+      - Note: `bpfman-csi` does not need to be updated.
+  - Run `cargo generate-lockfile`
+  - Update the bpfman version in the [bpfman/examples/Makefile][examples-makefile]:
+    - VERSION ?= x.y.z
+  - Add a new `bpfman/examples/config/v0.x.y/` and
+    `bpfman/examples/config/v0.x.y-selinux/` config directory for the release
+    version by copying the latest release directory and running a search for
+    the current release and replace with the new release.
+  - Add new example config directories for any new examples added since the
+    last release.
+  - Update dispatcher tags.
+    - Modify the tag for `XDP_DISPATCHER_IMAGE` and `TC_DISPATCHER_IMAGE` in
+      `bpfman/src/lib.rs` from `latest` to the new release tag.
+    - Manually add the new release tag to the latest version of the following
+      dispatcher images:
+      - https://quay.io/repository/bpfman/xdp-dispatcher
+      - https://quay.io/repository/bpfman/tc-dispatcher
```

Comment:
These aren't links in the generated docs. Not sure if they need to be, but wouldn't hurt.

Reply:
Updated.